enterprise/frontend/src/embedding-sdk/cli/run.ts

```diff
@@ -0,0 +1,64 @@
+import { ANONYMOUS_TRACKING_INFO } from "embedding-sdk/cli/constants/messages";
+import { printEmptyLines, printInfo } from "embedding-sdk/cli/utils/print";
+
+import {
+  addEmbeddingToken,
+  checkIsDockerRunning,
+  createApiKey,
+  generateCredentials,
+  generateCodeSample,
+  pollMetabaseInstance,
+  setupMetabaseInstance,
+  showMetabaseCliTitle,
+  startLocalMetabaseContainer,
+  checkIfReactProject,
+  checkSdkAvailable,
+} from "./steps";
+import type { CliState } from "./types/cli";
+
+export const CLI_STEPS = [
+  ["showMetabaseCliTitle", showMetabaseCliTitle],
```

Comment:
array is a convenient structure, but it lacks semantic. I was only until I found the first argument was used as `stepId` that I understood it wass an ID of some sort.

I think using an object is way more explicit. e.g. 

```suggestion
  { id: "showMetabaseCliTitle", executeStep: showMetabaseCliTitle },
```